ProfileManager.js

```diff
@@ -832,12 +849,21 @@ export default class ProfileManager {
     }
     const ephemeralSigner = await this._getAgentSigner(
       {profileAgentId: profileAgent.id, useEphemeralSigner: true});
+    let expires;
+    if(agentSigner && agentSigner.capability) {
+      expires = agentSigner.capability.expires;
+    } else {
+      const now = Date.now();
+      const ttl = this.ttl;
+      expires = new Date(now + ttl).toISOString();
```

Comment:
We can avoid the extra var ... but this should change to `this.zcapTtl` as well per the namespacing comment above:

```suggestion
      expires = new Date(now + this.zcapTtl).toISOString();
```